library/MedEx/API.php

```diff
@@ -1054,27 +1137,21 @@ private function addRecurrent($appt, $interval, $timing, $timing2, $M_group = "R
             $start_date = $timing;
             $stop_date = $timing2;
         }
-        $this->MedEx->logging->log_this("start: ".$start_date." -- stop: ".$stop_date);
+        
         //foreach date between curdate + timing and curdate + timing2 excluding dates excluded in recurring
         $hits = $this->MedEx->events->calculateEvents($appt, $start_date, $stop_date);
-        $this->MedEx->logging->log_this($hits);
-        $this->MedEx->logging->log_this("above are the hits for pc_eid: " . $appt['pc_eid']."\n\n
-        &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&\n
-        Now Convert them to real appt slots for MedEx\n");
-
+        
         //any dates that match need to be spawned from recurrent and made to live on their own.
-        $oldRecurrspec = unserialize($appt['pc_recurrspec'], ['allowed_classes' => false]);
-
+        $oldRecurrspec = unserialize($appt['pc_recurrspec']);
```

Comment:
revert above line change or is a big security vulnerability (let me know if not working with the change)

Reply:
No problemo.